content/education/teach-and-learn-with-github-education/about-campus-experts.md

```diff
@@ -9,3 +9,5 @@ versions:
 ---
 
 Learn public speaking skills, technical writing, community leadership, and software development skills as a {% data variables.product.prodname_dotcom %} Campus Expert. For more information, see "[Campus Experts](https://education.github.com/students/experts)."
```

Comment:
This link to https://education.github.com/students/experts now appears in the new text, so we should remove it from here to avoid duplication:

```suggestion
Learn public speaking skills, technical writing, community leadership, and software development skills as a {% data variables.product.prodname_dotcom %} Campus Expert. 
```